packages/core/src/codewhisperer/service/securityScanHandler.ts

```diff
@@ -59,7 +59,7 @@ export async function listScanResults(
     })
     codeScanIssueMap.forEach((issues, key) => {
         projectPaths.forEach(projectPath => {
-            const filePath = path.join(projectPath, '..', key)
+            const filePath = path.join(projectPath, key.split('/').slice(1).join('/'))
```

Comment:
This seems non-obvious, might be worth a one-line comment so that others in the future know about this quirk if they are changing the code